flight/mcu/src/mcu_ac/main.cpp

```diff
@@ -51,29 +59,45 @@ sensorDataStruct_t sensorData;
 
 FSM_State rocketState = STATE_INIT;
```

Comment:
Is this used at all? I think it's initialized already in sensor data struct. @GigaVoltFlash let me know if I can delete this line

Reply:
removing